stac_fastapi/api/stac_fastapi/api/app.py

```diff
@@ -363,14 +370,44 @@ def register_core(self) -> None:
 
     def add_health_check(self) -> None:
         """Add a health check."""
-        mgmt_router = APIRouter(prefix=self.app.state.router_prefix)
 
-        @mgmt_router.get("/_mgmt/ping")
         async def ping():
-            """Liveliness/readiness probe."""
+            """Liveliness probe."""
             return {"message": "PONG"}
 
-        self.app.include_router(mgmt_router, tags=["Liveliness/Readiness"])
+        self.app.router.add_api_route(
+            name="Ping",
+            path="/_mgmt/ping",
+            response_model=Dict,
+            responses={
+                200: {
+                    "content": {
+                        MimeTypes.json.value: {},
+                    },
+                },
+            },
+            response_class=self.response_class,
+            methods=["GET"],
+            endpoint=ping,
+            tags=["Liveliness/Readiness"],
+        )
```

Comment:
I don't think we needed to create a `mgmt_router`. I changed the logic to match what we do for the other endpoints (using `router.add_api_route`)